packages/graphql-docs-generator/src/generator/getFields.ts

```diff
@@ -61,3 +61,10 @@ export default function getFields(
     hasBody: !!(fields.length || fragments.length),
   };
 }
+
+function adjustDepth(fieldName, depth) {
+  if (fieldName == 'aggregateItems') {
+    return depth + 1;
```

Comment:
can the depth be increased indefinitely?  Would need to check for some upper bound.

Reply:
I've added a statement to check for the typename. Since 'SearchableAggregateResult' type is generated by CLI V2 transformer, we can be sure this doesn't go through infinite loop.

Reply:
in the case of isGraphQLAggregateField(field) is true and depth >= maxDepth, we may need to either return depth or throw errorr.  Currently it'll return depth-1 which may not be what's intended.

Reply:
Same question as above. If it is the aggregate field but the depth reaches maxDepth, we cannot return `depth-1` here. I suggest throwing an error.